vitrina/projects/urls.py

```diff
@@ -38,6 +39,36 @@
         ProjectPermissionsView.as_view(),
         name="project-permissions",
     ),
+    path(
+        "projects/<int:pk>/client/",
+        ClientListView.as_view(),
+        name="project-clients",
+    ),
+    path(
+        "projects/<int:pk>/client/add/",
+        ClientCreateView.as_view(),
+        name="project-clients-create"
+    ),
+    path(
+        "projects/<int:pk>/client/<int:client_id>/change/",
+        ClientUpdateView.as_view(),
+        name="project-clients-update"
+    ),
+    path(
+        "projects/<int:pk>/client/<int:client_id>/",
+        ClientDetailView.as_view(),
+        name="project-clients-detail",
+    ),
+    path(
+        "projects/<int:pk>/client/<int:client_id>/scopes/add/",
+        ClientScopeCreateView.as_view(),
+        name="project-clients-scopes-create"
+    ),
+    path(
+        "projects/<int:pk>/client/<int:client_id>/scopes/<int:scope_id>/toggle/",
+        ClientScopeToggleView.as_view(),
+        name="project-clients-scopes-toggle",
```

Comment:
To suggest that this is done for one object only:

```suggestion
        name="project-clients-scopes-detail-toggle",
```